systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTemplates.java

```diff
@@ -42,70 +36,54 @@ private KafkaTemplates() {}
     }
 
-    public static KafkaBuilder kafkaEphemeral(String clusterName, int kafkaReplicas, int zookeeperReplicas) {
+    public static KafkaBuilder kafkaEphemeral(String namespaceName, String clusterName, int kafkaReplicas, int zookeeperReplicas) {
         if (Environment.isKafkaNodePoolsEnabled()) {
             if (Environment.isKRaftModeEnabled()) {
-                return kafkaEphemeralKRaft(clusterName, kafkaReplicas);
+                return kafkaEphemeralKRaft(namespaceName, clusterName, kafkaReplicas);
             }
-            return kafkaEphemeralNodePools(clusterName, kafkaReplicas, zookeeperReplicas);
+            return kafkaEphemeralNodePools(namespaceName, clusterName, kafkaReplicas, zookeeperReplicas);
         } else {
-            return kafkaEphemeralWithoutNodePools(clusterName, kafkaReplicas, zookeeperReplicas);
+            return kafkaEphemeralWithoutNodePools(namespaceName, clusterName, kafkaReplicas, zookeeperReplicas);
         }
     }
 
-    public static KafkaBuilder kafkaEphemeralWithoutNodePools(String clusterName, int kafkaReplicas, int zookeeperReplicas) {
-        Kafka kafka = getKafkaFromYaml(TestConstants.PATH_TO_KAFKA_EPHEMERAL_CONFIG, false);
-        return defaultKafkaWithoutNodePools(kafka, clusterName, kafkaReplicas, zookeeperReplicas);
+    public static KafkaBuilder kafkaEphemeralWithoutNodePools(String namespaceName, String clusterName, int kafkaReplicas, int zookeeperReplicas) {
+        return defaultKafkaWithoutNodePools(namespaceName, clusterName, kafkaReplicas, zookeeperReplicas);
     }
 
-    public static KafkaBuilder kafkaEphemeralNodePools(String clusterName, int kafkaReplicas, int zookeeperReplicas) {
-        Kafka kafka = getKafkaFromYaml(PATH_TO_KAFKA_PERSISTENT_NODE_POOLS_EXAMPLE, true);
-
-        kafka.getSpec().getZookeeper().setStorage(null);
-
-        return defaultKafkaNodePools(kafka, clusterName, kafkaReplicas, zookeeperReplicas)
-            .editSpec()
-            .editZookeeper()
-                // the NodePools (in ZK mode) example contains persistent storage for ZK, so we need to specify the
-                // ephemeral storage
-                .withNewEphemeralStorage()
-                .endEphemeralStorage()
-            .endZookeeper()
-            .endSpec();
+    public static KafkaBuilder kafkaEphemeralNodePools(String namespaceName, String clusterName, int kafkaReplicas, int zookeeperReplicas) {
+        return defaultKafkaNodePools(namespaceName, clusterName, kafkaReplicas, zookeeperReplicas);
     }
 
-    public static KafkaBuilder kafkaEphemeralKRaft(String clusterName, int kafkaReplicas) {
-        Kafka kafka = getKafkaFromYaml(PATH_TO_KAFKA_EPHEMERAL_KRAFT_EXAMPLE, true);
-        return defaultKafkaKRaft(kafka, clusterName, kafkaReplicas);
+    public static KafkaBuilder kafkaEphemeralKRaft(String namespaceName, String clusterName, int kafkaReplicas) {
+        return defaultKafkaKRaft(namespaceName, clusterName, kafkaReplicas);
```

Comment:
I am fine with `clusterName` here, but I would prefer `kafkaClusterName` as well

Reply:
It also applies to all following occurrences